metadata.json

```diff
@@ -26,7 +26,7 @@
     },
     {
       "name": "katello/certs",
-      "version_requirement": ">= 9.0.0 < 11.0.0"
+      "version_requirement": ">= 11.0.0 < 12.0.0"
```

Comment:
Why is version 11 required? Looking at the release log all the changes were in areas we don't touch in this module so normally I would expect it to be `>= 9.0.0 < 12.0.0`.